paymentsheet/build.gradle

```diff
@@ -139,6 +139,9 @@ android {
 
     kotlinOptions {
         freeCompilerArgs += ["-opt-in=kotlinx.coroutines.FlowPreview"]
+        if (gradle.ext.isCi) {
+            kotlinOptions.allWarningsAsErrors = true
+        }
```

Comment:
It’ll be frustrating to only learn about warnings when CI fails, but enforcing it locally could also become tedious. Any thoughts on the tradeoffs?

Reply:
I've been annoyed when things fail to compile when I'm just messing with stuff locally. I think of it more like ktlint and detekt. 

It would be easy to update to set a gradle.properties locally or something if you prefer it the other way.

Reply:
Ok, let’s give it a try! I’m good with this being merged.